src/app/about/about.component.spec.ts

```diff
@@ -1,14 +1,20 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { AboutComponent } from './about.component';
+import { DataService } from './../services/data.service';
+import { Http } from '@angular/http';
 
```

Comment:
does x prevent this from running? If so, that's awesome. I used fdescribe all the time but didn't know about x.